lib/testmap.py

```diff
@@ -27,16 +27,16 @@
     },
     'cockpit-project/cockpit': {
         'main': [
-            'fedora-34/container-bastion',
+            'fedora-35/container-bastion',
             'debian-stable',
             'debian-testing',
             'ubuntu-2004',
             'ubuntu-stable',
             'fedora-34',
-            'fedora-34/devel',
-            'fedora-34/firefox',
-            'fedora-34/mobile',
-            'fedora-35',
+            TEST_OS_DEFAULT,
+            TEST_OS_DEFAULT + '/devel',
+            TEST_OS_DEFAULT + '/firefox',
+            TEST_OS_DEFAULT + '/mobile',
```

Comment:
...and for these three, please use f-strings.

Reply:
Done.